products/compute/terraform.yaml

```diff
@@ -470,7 +469,40 @@ overrides: !ruby/object:Overrides::ResourceOverrides
     custom_code: !ruby/object:Provider::Terraform::CustomCode
       post_create: templates/terraform/post_create/labels.erb
   GlobalForwardingRule: !ruby/object:Overrides::Terraform::ResourceOverride
-    exclude: true
+    examples:
+      - !ruby/object:Provider::Terraform::Examples
+        name: "global_forwarding_rule_http"
+        primary_resource_id: "default"
+        vars:
+          forwarding_rule_name: "global-rule"
+          http_proxy_name: "target-proxy"
+          backend_service_name: "backend"
+    properties:
+      backendService: !ruby/object:Overrides::Terraform::PropertyOverride
+        exclude: true
+      creationTimestamp: !ruby/object:Overrides::Terraform::PropertyOverride
+        exclude: true
+      id: !ruby/object:Overrides::Terraform::PropertyOverride
+        exclude: true
+      IPAddress: !ruby/object:Overrides::Terraform::PropertyOverride
+        default_from_api: true
+      IPProtocol: !ruby/object:Overrides::Terraform::PropertyOverride
+        diff_suppress_func: 'caseDiffSuppress'
+        default_from_api: true
+      loadBalancingScheme: !ruby/object:Overrides::Terraform::PropertyOverride
+        exclude: true
+      network: !ruby/object:Overrides::Terraform::PropertyOverride
+        exclude: true
+      portRange: !ruby/object:Overrides::Terraform::PropertyOverride
+        diff_suppress_func: 'portRangeDiffSuppress'
+      ports: !ruby/object:Overrides::Terraform::PropertyOverride
```

Comment:
Are these fields in the API, or just the shared FR / GFR proto object? If they're not a real part of the API, we should remove them from api.yaml or exclude them across all providers.

Reply:
They are part of the API, just not included in the current version of TF. They got added to api.yaml when Alex inlined the FR properties, and I just didn't feel like deleting them to reinclude them later. My second PR re-includes them in terraform.yaml